wiki/Contests/PHFC/2024/en.md

```diff
@@ -0,0 +1,67 @@
+---
+tags:
+  - phfc
+  - phfc 2024
+---
+
+# osu! Philippines Fanart Contest 2024 — "Kasiyahan sa Kapaskohan"
+
+![](img/logo.png)
+
+The **osu! Philippines Fanart Contest 2024 — "Kasiyahan sa Kapaskohan"** (***PHFC2024***) is a fanart contest made for the Philippine osu! community featuring its four mascots. It is themed after the Christmas season.
+
+## Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Submission period | 2024-10-11/2024-11-08 (15:59 UTC) |
+| Entry screening | 2024-11-08/2025-11-15 |
+| Public voting | 2024-11-15/2024-11-29 |
+
+## Prizes
+
+The contest offers a minimum prize pool worth PHP 2,500.00. (*Amount is currently tentative.*)
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 50% of the prize pool, unique profile badge (*proposed*) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 35% of the prize pool |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 15% of the prize pool |
+
+## Organisation
+
+The osu! Philippines Mascots Fanart Contest 2024 is run by the following staff:
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595), ::{ flag=PH }:: [SurfChu85](https://osu.ppy.sh/users/4469895), ::{ flag=PH }:: [topecnz](https://osu.ppy.sh/users/2103927) |
+| Contest moderator | ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595), ::{ flag=CN }:: [Sakura006](https://osu.ppy.sh/users/10365024), ::{ flag=PH }::  [topecnz](https://osu.ppy.sh/users/2103927) |
+| Developer | ::{ flag=PH }:: [-Ichinose](https://osu.ppy.sh/users/12635292), ::{ flag=PH }:: [topecnz](https://osu.ppy.sh/users/2103927) |
+| Statistician | ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595), ::{ flag=PH }:: [SurfChu85](https://osu.ppy.sh/users/4469895) |
+| Wiki editor | ::{ flag=PH }:: [SurfChu85](https://osu.ppy.sh/users/4469895) |
+
+## Links
+
+- Submission form
+- Discussion thread
+- [osu! Philippines Discord server](https://discord.gg/officialosuph)
+
+## Rulesets
+
+- By submitting your artwork to this contest, you give osu! Philppines the permission to use it for any promotional material for its activities, not limited to events, promotions, and the like. You will retain all of the rights to your work and you are permitted to sell or distribute it.
```

Comment:
```suggestion
- By submitting your artwork, you give osu! Philppines the permission to use it for any promotional material for its activities including (but not limited to) events, promotions, and the likes. You will retain all of the rights to your work, and you will be permitted to sell or distribute it.
```

Reply:
"like" in "and the like" doesn't take a plural form